cmd/admin.go

```diff
@@ -68,6 +72,7 @@ Example:
 	}
 
 	deleteAliasCommand.Annotations["<name>"] = "The name of the alias to delete"
+	deleteAliasCommand.Flags().BoolVar(&nsOpts.noPrompt, "no-prompt", false, "Disable prompt to bypass interactive UI.")
```

Comment:
I just realized that the [deleteNamespaceAlias](https://github.com/CircleCI-Public/circleci-cli/blob/master/cmd/namespace.go#L98) function doesn't actually do a `if opts.NoPrompt { ... }` check. Do you think theres value in adding it for non-integration test purposes? 🤔

Reply:
Yes, thanks for noticing that. Done.